### DISCUSSION TITLE
Issue with Output Parameters in GEOPHIRES-X example1: How to Obtain LCOE or LCOH?

### DISCUSSION THREAD

**Discussion by huhanyu** [NatLabRockies/GEOPHIRES-X#89] [Q&A] [answered]
When running example1 in GEOPHIRES-X, the default output is the Net Present Value (NPV) of the project. However, I am interested in obtaining the Levelized Cost of Energy (LCOE) or Levelized Cost of Heat (LCOH) instead. I have searched through the documentation but have not found any instructions on how to modify the output to include these parameters. Could you please guide me on how to proceed to obtain LCOE or LCOH in this scenario? Your assistance would be greatly appreciated.

**Comment by softwareengineerprogrammer** [comment, marked as answer]
@malcolm-dsider or @kfbeckers I believe LCOE and LCOH are output as "Electricity breakeven price" and "Direct-Use heat breakeven price" respectively, can you confirm?

@huhanyu In the meantime, if you can share your input file I can look to see if there are any issues with how GEOPHIRES is processing it.

**Reply by huhanyu** [reply]
Thanks for your reply, previously when I ran the Beckers_et_al_2023_Tabulated_Database_Uloop_water_heat.txt input file in the tests folder the output was LCOH, when I ran the example1.txt input file the output was no longer LCOH it became Electricity breakeven price, so I had a misunderstanding. Now I got it, maybe the output style of EGS and AGS should be harmonized :)
![1c65ec809c9cebcf0f530288cb5f3a59](https://github.com/NREL/GEOPHIRES-X/assets/22694173/449e950e-eb27-444f-a38a-dfe205db5477)
![1ada48a6a29094f5bd4c978b672b7d3f](https://github.com/NREL/GEOPHIRES-X/assets/22694173/c11831e2-ae86-403c-a302-6cc0e6e5f14a)

**Comment by kfbeckers** [comment]
Yes, that is correct. LCOE is stored in model.economics.LCOE and LCOH is
stored in model.economics.LCOH.

Regards,
Koenraad
